functorch/functorch/_src/aot_autograd.py

```diff
@@ -317,6 +317,11 @@ def rearrange(tensor_args, static_args, static_argnums):
 
     return args
 
+def fake_signature(fn, nargs):
+    """FX gets confused by varargs, de-confuse it"""
+    argnames = ",".join(f"arg{i}" for i in range(nargs))
+    return eval(f"lambda {argnames}: fn({argnames})", {"fn": fn})
```

Comment:
I moved this all the way into `make_fx` in https://github.com/pytorch/pytorch/pull/83210 so you don't need it here anymore